lynx/common/object.py

```diff
@@ -18,9 +18,8 @@ class Object(Entity):
     position: Vector = Vector(0, 0)
     additional_positions: List[Vector] = field(default_factory=list) 
     state: str = ""
-    walkable: bool = False
     tick: str = ""
     on_death: str = ""
     owner: str = ""
-    pickable: bool = False
-    pushable: bool = False
+    inventory: List[int] = field(default_factory=list)
```

Comment:
I think inventory should be implemented in separate pull request, if not Jira issue